SCons/Environment.xml

```diff
@@ -3094,16 +3094,22 @@ they are not added and included in the returned list.
 <arguments>(arg)</arguments>
 <summary>
 <para>
-Returns a list of file names or other objects.
 If <parameter>arg</parameter> is a string,
-it will be split on strings of white-space characters
-within the string,
-making it easier to write long lists of file names.
-If <parameter>arg</parameter> is already a list,
-the list will be returned untouched.
+splits on whitespace and returns a list of
+strings without whitespace.
+This mode is the most common case,
+and can be used to split a list of filenames
+(for example) rather than having to type them as a
+list of individually quoted words.
+If <parameter>arg</parameter> is a list or tuple
+returns <parameter>arg</parameter> unchanged.
+Use this form to specify strings where whitespace
```

Comment:
? I'd drop the reference to whitespace as it's not always relevant.

Reply:
The main reason to call Split() is to ensure that you have a list (or tuple) of items to process. I think adding more detail on various usages isn't adding value, but likely confusing the usage.

One note is as you've included if you pass a list you get a list, if you pass a tuple, you will get a tuple and not a list. Perhaps just if you pass a tuple you will not get a list, but a tuple returned.  All other times you'll get a list.

Reply:
I continue to fail to see why it even makes sense to call Split on something that doesn't need splitting - this is one of those scons overdesign-things.  So I wanted to try to leave some hint.

Reply:
Because when it's called the caller may not know if the object is a sequence or not. So rather than check that all over the place, you can just call Split() on it and know what comes out the other side is the expected sequence.

Reply:
> One note is as you've included if you pass a list you get a list, if you pass a tuple, you will get a tuple and not a list. Perhaps just if you pass a tuple you will not get a list, but a tuple returned. All other times you'll get a list.

You get back a list, but there's a difference between passing a list and passing something else: if I pass a `Foo` object named `foo` I get back `[foo]`, while I passed a list object named `foo` I get back `foo`. Anyway, the existing wording already says that, the change to this part is calling out tuple as returning tuple, since that's what the impl. already does.